File: Python/Manual 04/Question03.py
tens = int(input("Enter Tens Limit: "))
ones = int(input("Enter Ones Limit: "))

if tens > 9 or tens < 1 or ones > 9 or ones < 0:
    print("Invalid Input")
else:
    i = 1
    while i <= tens:
        j = 0
        while j <= ones:
            print(str(i) + str(j))
            j += 1
        i += 1

File: JavaScript/Manual 02/Question03.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question("Enter your percentage: ", function (percentage) {
    switch (true) {
        case (percentage >= 80):
            console.log("A+");
            break;
        case (percentage >= 70):
            console.log("A");
            break;
        case (percentage >= 60):
            console.log("B");
            break;
        case (percentage >= 50):
            console.log("C");
            break;
        case (percentage >= 40):
            console.log("D");
            break;
        case (percentage < 40):
            console.log("F");
            break;
        default:
            console.log("Invalid Input");
            break;
    }

    rl.close()
})